test/unit3/test_inheritance.py

```diff
@@ -109,8 +109,8 @@ def test_skulpt_bugs(self):
         self.assertTrue(isinstance(type, type))
         self.assertTrue(isinstance(object, object))
         self.assertTrue(isinstance(object, type))
-        # self.assertTrue(issubclass(type, object)) # commenting out until issubclass returns a pyBool
-        # self.assertFalse(issubclass(object, type))
+        self.assertTrue(issubclass(type, object))
+        self.assertFalse(issubclass(object, type))
 
```

Comment:
is travis bugging out? this change works fine on my home pc and can't be the cause of all the run time errors that travis found... ?

Reply:
something to not compatible with a recent commit to master - will investigate and get back